mdn/kuma
# Subscriptions Form button fix and tests

Changes:
- Fix bug in #6931, where Stripe modal couldn't open a second time. 
- Add some tests. Initially I wanted to cover the full component but it was taking up too much time (I didn't want to spend all day on writing a test!), so I committed what I had so far. 
- Reduce size of "Sorry!" heading
- Consolidate payments error messages 

To test: 
- Go to /payments and ensure making a subscription still works. 
- Ensure error messages still work and look ok: 
<img width="962" alt="sorry-text" src="https://user-images.githubusercontent.com/650/80819970-069be980-8ba4-11ea-8706-8c96a0969fce.png">


Closes #6931
